cms/settings/base.py

```diff
@@ -979,6 +979,11 @@
     "Page",
 )
 
+# Allowed prefixes for iframe visualisations
+IFRAME_VISUALISATION_EMBED_PREFIX_LIST = env.get(
+    "IFRAME_VISUALISATION_EMBED_PREFIX_LIST", ["*.ons.gov.uk", "*.onsdigital.uk"]
+)
```

Comment:
suggestion: given we get strings from the env, split it parse time

```suggestion
IFRAME_VISUALISATION_EMBED_PREFIX_LIST = env.get(
    "IFRAME_VISUALISATION_EMBED_PREFIX_LIST", "*.ons.gov.uk,*.onsdigital.uk"
).split(",")
```

Reply:
Now updated